banyand/stream/write.go

```diff
@@ -134,7 +139,7 @@ func (w *writeCallback) handle(dst map[string]*elementsInGroup, writeEvent *stre
 				tagFamilySpec.Tags[j].Type,
 				tagValue,
 			))
-			if tagFamilySpec.Tags[j].IndexedOnly {
+			if tagFamilySpec.Tags[j].IndexedOnly || entityMap[tagFamilySpec.Tags[j].Name] {
 				continue
 			}
 			tagFamilies[i].values = append(tagFamilies[i].values, encodeTagValue(
```

Comment:
As I mentioned in the measure's "write.go", merging encoding tag operations is better.